demos/projects/ESPRESSIF/adu/components/sample-azure-iot/transport_socket_esp32.c

```diff
@@ -117,7 +117,6 @@ SocketTransportStatus_t Azure_Socket_Connect( NetworkContext_t * pNetworkContext
         if( pNetworkContext != NULL )
         {
             esp_transport_close( pxEspSocketTransport->xTransport );
-            esp_transport_destroy( pxEspSocketTransport->xTransport );
             esp_transport_list_destroy(pxEspSocketTransport->xTransportList);
```

Comment:
@danewalton shouldn't this be fixed in all other implementations?

E.g.: 

https://github.com/Azure-Samples/iot-middleware-freertos-samples/blob/7ffec26278a89cc81ff1eb05e9053a71423c328d/demos/projects/ESPRESSIF/adu/components/sample-azure-iot/transport_socket_esp32.c#L120

[Full text search using esp_transport_delete.](https://github.com/Azure-Samples/iot-middleware-freertos-samples/search?q=esp_transport_destroy)

Reply:
Not yet. The other implementations don't use the list yet. So when we update those, we can go ahead and use this idea.